addons/im_livechat/static/src/embed/common/message_patch.xml

```diff
@@ -8,8 +8,8 @@
             <t t-else="">$0</t>
         </xpath>
         <xpath expr="//*[hasclass('o-mail-Message-textContent')]" position="after">
-            <ul class="p-0 m-0" t-if="props.message.chatbotStep?.answers and !props.message.chatbotStep.selectedAnswer">
-                <li class="list-unstyled" t-foreach="props.message.chatbotStep?.answers" t-as="answer" t-key="answer.id">
+            <ul class="p-0 m-0" t-if="props.message.chatbotStep?.answer_ids and !props.message.chatbotStep.selectedAnswer">
```

Comment:
Already before this PR but this check is useless as it is an array, it will always be true. It should check the length.

Reply:
I guess it doesn't hurt to fix this.